spec/CloudCode.spec.js

```diff
@@ -3727,11 +3734,18 @@ describe('saveFile hooks', () => {
         foo: 'bar',
       },
     };
+    // Get the actual config values that will be used
+    const expectedConfig = {
```

Comment:
Can you import this from the test suite where it's defined?

Reply:
I added it but i still stripped it down because previously the config had been modified through the process or something. We'll see if it passes CI, but Maybe the check just needs to be loosened here.